reconcile/cli.py

```diff
@@ -219,6 +220,11 @@ def init_log_level(log_level):
     logging.basicConfig(format=format,
                         datefmt='%Y-%m-%d %H:%M:%S',
                         level=level)
+    syslog = SysLogHandler(address=('localhost', 5014))
```

Comment:
make host & port configurable from `config.toml`

Reply:
What happens if there is no syslog server like in dev env?

Reply:
Syslog goes through UDP connect. There are any errors or warnings for no Syslog server.

Reply:
Do you mean to say there will be no errors?

Reply:
Yes.